README.md

```diff
@@ -1 +1,71 @@
-# Awesome-books
\ No newline at end of file
+# 
+Awesome Books.
+
+For this repository, we've created this to manage book inventory. User can add and remove book from the the UI.
+
+I used ESlint to manage the files for the front-end, using loaders for HTML, CSS, and JavaScript files, in order to fix some special thing at the moment to use the front-end in a server. Feel free to suggest more additional features for this repository, or if you want to use it, feel free to do that as well.
+
+![alt text](https://github.com/micheaol/Awesome-books/blob/books-with-object/assets/images/project-img.PNG)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+    -HTML
+    -CSS
+    -JavaScript
+- Frameworks
+- Technologies used
+    -HTML
+    -CSS
+    -JS
+
+## Live Demo
+
+<!-- [Live Demo Link](https://micheaol.github.io/Portfolio-project/)
+[Live Demo Link](https://micheaol-portfolio.netlify.app/) -->
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@micheaol](https://github.com/micheaol)
+- Twitter: [@micheaol](https://twitter.com/micheaol)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/micheaol)
+
+👤 **Author2**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author1** and **Author2** with your respective **full names**, so as to make your ReadMe appear more professional